.github/workflows/release.yml

```diff
@@ -0,0 +1,109 @@
+# This workflow will release the packages.
+
+name: 🚀 Release - 🤖 Android SDKs
+
+on:
+  workflow_dispatch:
+    inputs:
+      sdkType:
+        type: choice
+        description: Choose which sdk you want to release
+        default: 'android'
+        options:
+        - android
+        - android-core
+      version:
+        type: choice
+        description: Choose which version to bump before release
+        default: 'Patch'
+        options:
+        - Major
+        - Minor
+        - Patch
+
+env:
+    # GH_TOKEN: ${{ secrets.RELEASE_BOT_TOKEN }}
```

Comment:
Is commenting out this intentional?

Reply:
removed comments

Reply:
Fixed with https://github.com/asgardeo/mobile-ui-sdks/pull/1/commits/67ea36cc9fef864e59221f887574afe50546eea3